### DISCUSSION TITLE
Countdown timer functionality

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-future#373] [Feature Requests]
https://wordpress.org/support/topic/countdown-timer-functionality/

> I’d pay for a “Pro” version of this plugin that includes countdown timer functionality.
> 
> Instead of a mere [postexpirator] shortcode that shows the expiration time/date, I’d love to see either a new shortcode that counts down to the expiration date, or add a parameter that “upgrades” the date/time display to a countdown timer.
> 
> The styling doesn’t need to be fancy, in my opinion
>

**Comment by rizaardiyanto1412** [comment]
Another request:

> Hello, is it possible to display a countdown ?
> 
> In example display “99 days left” or “Post expire in 99 Days”
> 
> thank you

https://wordpress.org/support/topic/display-countdown-in-days/
